Documentation.md

```diff
@@ -426,3 +426,239 @@ You should see the new changes. If there are dependencies errors, please install
 To stop the server press \*control c\*\* together.
 
 That's it! You have completed setting up the express node server backend.
+
+## API Routes
```

Comment:
This is the outdated documentation file. Documentation for using the CLI should be placed in its own file that can be referenced from other documents.